try:
    print("test")
except:
    print("except節に何も書かないとエラーになる")
